src/templates/views/payment.hbs

```diff
@@ -1,39 +1,628 @@
+        <option value="WF">Wallis and Furuna Islands</option>
+        <option value="EH">Western Sahara</option>
+        <option value="YE">Yemen</option>
+        <option value="ZM">Zambia</option>
+        <option value="ZW">Zimbabwe</option>
+      </select>
+
+      <label for="billing-postal-code">ZIP or postal code</label>
+      <input required autocomplete="postal-code" id="billing-postal-code" name="billing-postal-code">
+
+    </section>
+
+    <section id="shipping">
+
+      <h2>Shipping address</h2>
 
-    <label for="SCity">City</label>
-    <input name="ship-city" required id="SCity" placeholder="City" autocomplete="shipping city">
+      <input id="shipping-same" name="shipping-shipping-same" type="checkbox">
+      <label for="shipping-shipping-same">Same as billing address</label>
 
-    <label for="SZip">Zip / Postal code</label>
-    <input name="ship-zip" required id="SZip" placeholder="00000" autocomplete="shipping postal-code">
+      <label for="shipping-honorific-prefix">Title</label>
+      <input required autocomplete="honorific-prefix" id="shipping-honorific-prefix" name="shipping-honorific-prefix">
 
-    <label for="SCountry">Country</label>
-    <input name="ship-country" required id="SCountry" placeholder="Country" autocomplete="shipping country">
+      <label for="shipping-given-name">First name</label>
+      <input required autocomplete="given-name" id="shipping-given-name" name="shipping-given-name">
 
-    <h3>Payment</h3>
-    <label for="CCName">Name on card</label>
-    <input name="ccname" id="CCName" required placeholder="Full Name" autocomplete="cc-name">
+      <label for="shipping-family-name">Last name</label>
+      <input required autocomplete="family-name" id="shipping-family-name" name="shipping-family-name">
 
-    <label for="CCNum">Card Number</label>
-    <input name="cardnumber" id="CCNum" required autocomplete="cc-number">
+      <label for="shipping-address-line1">Address 1</label>
+      <input required autocomplete="address-line1" id="shipping-address-line1" name="shipping-address-line1">
 
-    <label for="CCCVC">CVC</label>
-    <input name="cvc" id="CCCVC" required autocomplete="cc-csc">
+      <label for="shipping-address-line2">Address 2</label>
+      <input autocomplete="address-line2" id="shipping-address-line2" name="shipping-address-line2">
 
-    <label for="CCExp">Expiry</label>
-    <input name="cc-exp" id="CCExp" required placeholder="MM-YYYY" autocomplete="cc-exp">
+      <label for="shipping-locality">City or town</label>
+      <input required autocomplete="locality" id="shipping-locality" name="shipping-locality">
+
+      <label for="shipping-state">State or region</label>
+      <input autocomplete="state" id="shipping-state" name="shipping-state">
+
+      <label for="shipping-country">Country</label>
+      <select requred autocomplete="country" id="shipping-country" name="shipping-country">
+        <option value="AF">Afghanistan</option>
```

Comment:
Consider making it a partial to avoid repetition.

Reply:
Excellent idea! 

I was groaning at this repetition, considering JavaScript options, etc.

Done.